src/Canvanoid.js

```diff
@@ -17,39 +17,93 @@ export default class Canvanoid {
 					  delta: null }
 
 	    this.messagePanel = null;
-		this.score = null;
+		this.instructionsPanel = null;
+		this.scorePanel = null;
 
 		this.state = null; 
 		this.board = null;
 		this.balls = null;
 		this.vaus = null; 
+		this.pause = null;
 	}
 	
 	start() {	
+		// Game Elements
 		this.state = new State();
```

Comment:
Sirikon located an error during his previous revision for which GitHub would add extra spaces to the code for no apparent reason. He theorized that my tabbing set up might be somehow mixing up tabs and spaces, and that's definitely something I must look after.
I usually use four spaces (one tab) because, as far as I know, that's the standard, but I have no problem using two spaces instead.
It seems it happened again, so I will look more in depth into it. Last time it happened only to two lines, but I'm seeing here it just went crazy.
Thank you for noticing!